hikey.mk

```diff
@@ -15,6 +15,9 @@ CFG_SW_CONSOLE_UART ?= 3
 # eMMC flash size: 8 or 4 GB [default 8]
 CFG_FLASH_SIZE ?= 8
 
+# URL to images
+NVME_IMG_URL=https://builds.96boards.org/releases/hikey/linaro/binaries/latest/nvme.img
```

Comment:
Can we get a fixed version for this?

Reply:
nvme.img is essentially a never-changing empty file (all 0s) used to clear the nvme partition. As such, it might be better to keep it at 'latest' to avoid a specific version (links) getting archived or removed.

Reply:
Then the command should be: `dd if=/dev/zero of=tutu bs=1K count=128`, not `wget ...`.

BTW, it's weird, I have an old `nvme.img` which is far from being all 0s:
```
$ strings nvme.img 
SWVERSI
T8300
BOARDID
MACADDR
WEBPWD
IMEI
IMEI
RSV6
RSV7
RSV8
...
```

I wonder what its purpose is.

Reply:
Can change to `dd` if that's preferred?

`nvme` partition is used to store uefi non-volatile variables, so your copy might be for a time when some of these variables were required on boot, but not anymore.

Reply:
Thanks for the info.
I definitely prefer `dd`, and if you could add a comment saying 'nvme partition is used to store UEFI non-volatile variables` that would be great!

Reply:
You're welcome!
I pushed a separate commit for changing `wget` to `dd` and adding the comment. Please check if it's ok. Thanks!

Reply:
On 2nd thought, maybe a separate commit is not required. Lemme know if so so I can squash it later.